frontend/src/metabase/visualizations/components/TableInteractive.jsx

```diff
@@ -362,7 +363,10 @@ export default class TableInteractive extends Component {
                         rowCount={rows.length}
                         rowHeight={ROW_HEIGHT}
                         cellRenderer={this.cellRenderer}
-                        onScroll={({ scrollLeft }) => onScroll({ scrollLeft })}
+                        onScroll={({ scrollLeft }) => {
+                            this.props.onActionDismissal()
```

Comment:
Feels a little wasteful to trigger `onActionDismissal` all the time during scroll but seems that there isn't anything like `onScrollStart` for react-virtualized `Grid` element 😕 So probably it's good as it currently is!